index.md

```diff
@@ -4,49 +4,151 @@ title: Software Engineering for Busy Experts
 ---
 # Software Engineering Best Practices for Busy Experts
 
-{% include opening_box.html content="I offer a series of seminars designed to help scientists, engineers and other domain experts to learn the most important software engineering best practices to help them write correct code more efficently." %}
+{% include opening_box.html content="I offer a course designed to help scientists, engineers and other domain experts to learn the most important software engineering best practices to help them write correct code more efficiently." %}
 
 **If your company has domain experts who are writing code, [get in touch](mailto:hi@tkiley.co.uk) today to increase both the reliability of the software you produce, as well as the speed at which you can develop it.**
 
-You can read more about what the course covers and how it is structured [here](#the-course).
+Writing code is now an _essential part of all science and technology_ research and development - yet many of the scientists and domain experts have _never received training on software engineering_ best practices, and are instead expected to figure it out. The result of this is slow software development and unreliable software.
 
-Get in touch today to discuss getting these seminars given to you. Email me [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) or ring me on 07948 284729.
+I can deliver a course that will teach your developers the _industry standard techniques_ to write more reliable code faster.
 
-> "Having Thomas come on site and give these talks massively improved our productivity"
+The course consists of _5 seminars_, with lots of time for questions. After the course, I would provide an _on-going relationship_ to help you apply the techniques.
```

Comment:
This doesn't feel like the most important thing to have at the top of the page. Ditto the next line.